### PR TITLE
Align terminology of Mutator Types #106

### LINKED ISSUE #106: Align terminology of Mutator Types
In the [handbook](https://github.com/stryker-mutator/stryker-handbook/blob/setup-handbook/mutator-types.md#supported-mutators) are the mutators described that are being used within the Stryker eco system.  Now there is a mismatch of the terminology being used in [documentation](https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.Core/README.md#boolean-mutator)/code of Stryker.NET.

Binary mutator -> Binary Operators
Boolean mutator -> Boolean Substitutions
PrefixUnaryStatements -> Update operators or Boolean Substitutions
PostfixUnaryStatements -> Update operators

### PR DESCRIPTION
**Pull request by joostas**
Fixes #106.

### CODE REVIEW

**Comment by richardwerkman** [comment]
While we are adressing the names of the mutator, we should make sure the naming is alligned. Some mutators are now called by the type of expressions they mutate, others are called a name and then "mutator". I think it's better when we call them all the same.

* Assignment Statements
* Checked Statements

@nicojs should the mutators be exactly the same on all platforms? And what is your thought about this?

**Comment by joostas** [comment]
Names were aligned based on this https://github.com/stryker-mutator/stryker-handbook/blob/master/mutator-types.md#supported-mutators.
As I understand you ask to remove Mutator postfix from `Assignment Mutator` and `Checked Mutatror`?

File: README.md [hunk 1]
```diff
@@ -35,12 +35,12 @@ Only compatible with .NET Core version 1.1+
...
 Right now, Stryker.NET supports the following mutators:
-- BinaryExpressionMutator
-- BooleanMutator
-- AssignmentStatementMutator
-- PrefixUnaryStatements
-- PostfixUnaryStatements
-- CheckedMutator
+- Binary Operators
+- Boolean Substitutions
+- Assignment Mutators
```
Comment (richardwerkman) [inline README.md]: This name says `Assignment Mutator` in the handbook.

File: src/Stryker.Core/README.md [hunk 2]
```diff
@@ -4,16 +4,16 @@ Stryker supports a variety of mutators, which are listed below. Do you have a su
...
 - [Mutators](#mutators)
-    - [Binary mutator](#binary-mutator)
-    - [Boolean mutator](#boolean-mutator)
-    - [Assignment mutator](#assignment-mutator)
-    - [PrefixUnaryStatements](#prefixunarystatements)
-    - [PostfixUnaryStatements](#postfixunarystatements)
-	- [Checked mutator](#checked-mutator)
+    - [Binary Operators](#binary-operators)
+    - [Boolean Substitutions](#boolean-substitutions)
+    - [Assignment Mutators](#assignment-mutators)
```
Comment (richardwerkman) [inline src/Stryker.Core/README.md]: Same here

File: src/Stryker.Core/README.md [hunk 3]
```diff
@@ -34,13 +34,14 @@ Stryker supports a variety of mutators, which are listed below. Do you have a su
...
 | Original | Mutated | 
 | ------------- | ------------- | 
-| `true` | `false` |
-| `false` | `true` |
+| `true`	| `false` |
+| `false`	| `true` |
+| `!`		| ` ` |
 
-## Assignment mutator
+## Assignment Mutators
```
Comment (richardwerkman) [inline src/Stryker.Core/README.md]: And here

**Comment by richardwerkman** [comment]
I opened an issue for alligning the names in the handbook. For now the names are ok here. Just one remark.

**Pull request merged**